src/app/file/svg_format.cpp

```diff
@@ -0,0 +1,161 @@
+// Aseprite
+// Copyright (c) 2018  Igara Studio S.A.
+//
+// This program is distributed under the terms of
+// the End-User License Agreement for Aseprite.
+//
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "app/doc.h"
+#include "app/file/file.h"
+#include "app/file/file_format.h"
+#include "app/file/format_options.h"
+#include "base/cfile.h"
+#include "base/file_handle.h"
+#include "doc/doc.h"
+
+namespace app {
+
+using namespace base;
+
+class SvgFormat : public FileFormat {
+  const char* onGetName() const override {
+    return "svg";
+  }
+
+  void onGetExtensions(base::paths& exts) const override {
+    exts.push_back("svg");
+  }
+
+  dio::FileFormat onGetDioFormat() const override {
+    return dio::FileFormat::SVG_IMAGE;
+  }
+
+  int onGetFlags() const override {
+    return
+      FILE_SUPPORT_SAVE |
+      FILE_SUPPORT_RGB |
+      FILE_SUPPORT_RGBA |
+      FILE_SUPPORT_GRAY |
+      FILE_SUPPORT_INDEXED |
+      FILE_SUPPORT_SEQUENCES;
+  }
+
+  bool onLoad(FileOp* fop) override;
+#ifdef ENABLE_SAVE
+  bool onSave(FileOp* fop) override;
+#endif
+};
+
+FileFormat* CreateSvgFormat()
+{
+  return new SvgFormat;
+}
+
+  bool SvgFormat::onLoad(FileOp* fop) { return false;}
+
+#ifdef ENABLE_SAVE
+
+bool SvgFormat::onSave(FileOp* fop)
+{
+  const Image* image = fop->sequenceImage();
+  unsigned char image_palette[256][4];
+  int x, y, c, r, g, b, a, alpha;
+  float opacity;
+  FileHandle handle(open_file_with_exception_sync_on_close(fop->filename(), "wb"));
+  FILE* f = handle.get();
+
+  if (image->pixelFormat() == IMAGE_INDEXED) {
+    for (y=0;y<256; y++) {
```

Comment:
Minor format change, missing whitespace: 
```suggestion
    for (y=0; y<256; y++) {
```